### PR TITLE
Create quickstart.yml

### PR DESCRIPTION
**Pull request by fivetran-joemarkiewicz**
Small PR to add quickstart.yml to be properly parsed by the Quickstart automation.

As this is just an addition of the quickstart.yml, there is no need for a new release or version bump.

### CODE REVIEW

**Review by fivetran-reneeli — APPROVED** [review]
lgtm

File: .quickstart/quickstart.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+database_key: marketo_database
+schema_key: marketo_schema
+
+dbt_versions: ">=1.3.0 <2.0.0"
+
+table_variables:
+  marketo__enable_campaigns:
+    - program
```
Comment (fivetran-reneeli) [inline .quickstart/quickstart.yml:8]:
```suggestion

```

Comment (fivetran-joemarkiewicz) [reply .quickstart/quickstart.yml:8]: @fivetran-reneeli program is intentionally included the variable is used in the [program staging model](https://github.com/fivetran/dbt_marketo_source/blob/e3480427b2067ff52ad9133fa4ec779d56e71d11/models/stg_marketo__program.sql#L1).

Comment (fivetran-reneeli) [reply .quickstart/quickstart.yml:8]: Ah thanks for pointing me to that 👍

**Review by fivetran-reneeli — APPROVED** [review]
looks good!

**Pull request merged**